src/ec/packages/ec-component-hero-banner/hero-banner.html.twig

```diff
@@ -0,0 +1,78 @@
+{% spaceless %}
+{#
+  Parameters:
+    - "type" (string) (default: 'default') Type of banner (can be 'default','image','image-shade','primary')
+    - "title" (string) (default: '') Title of banner
+    - "image" (string) (default: '') Image for banner (required for image banner type)
+    - "description" (string) (default: '') Description of banner
+    - "centered" (bool) (default: true) Define if banner should be centered
+    - "button" (associative array) (default: predefined structure) predefined structure for EC Button component
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# variables #}
+{% set _type = type|default('default') %}
+{% set _centered = centered|default(true) %}
+{% set _title = title|default('') %}
+{% set _description = description|default('') %}
+{% set _image = image|default('') %}
+
+{% set _button = {
+  variant: 'call',
+  icon: {
+    type: 'ui',
+    name: 'corner-arrow',
+    path: '',
+    size: 'xs',
+    transform: 'rotate-90'
+  },
+  label: '',
+  extra_classes: 'ecl-hero-banner__button'
+  }
+%}
+
+{% set _css_class = ['ecl-hero-banner ecl-hero-banner--'~_type]|join(' ') %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+{% if button is defined and button is not empty %}
```

Comment:
why do you need to do these modifications on button data?
As far as I see, the data is already set in the correct format in your story.js file, so you should just have to use it as is.

Reply:
I thought that some button parameters will be constant - I wanted to give the user the ability to change only those parameters that are necessary. But in this case, what you write, everything will be defined directly by the user, I also removed the unnecessary code.
Fixed in https://github.com/ec-europa/ecl-twig/pull/49/commits/97024e465b8650a3f7608ed55c200d09723e45b6